db/migrate/20230823083547_add_data_analyst_role_enum.rb

```diff
@@ -0,0 +1,21 @@
+class AddDataAnalystRoleEnum < ActiveRecord::Migration[7.0]
+  def up
+    execute <<-SQL
+      ALTER TYPE user_role ADD VALUE IF NOT EXISTS '#{Types::DATA_ANALYST_ROLE}';
+    SQL
+  end
+
+  def down
+    data_analysts = User.unscoped.where(role: Types::DATA_ANALYST_ROLE)
+    puts "#{data_analysts.size} Data Analyst(s) will be destroyed!"
+    data_analysts.destroy_all
```

Comment:
just wondering with this what is the implication if someone moves from caseworker to data analyst and they have done "work" on the system and then is able to be destroyed. Not sure it should matter as I think it is only reference to the ids of users that are used in events.

Reply:
Comment no longer applicable because decided to just not allow rollbacks from this day on.